src/analysis/exploration/biomassPrecursorCheck.m

```diff
@@ -1,7 +1,7 @@
-function [missingMets, presentMets,coupledMets] = biomassPrecursorCheck(model,checkCoupling)
+function [missingMets, presentMets,coupledMets, missingCofs, presentCofs] = biomassPrecursorCheck(model,checkCoupling)

```

Comment:
Given that #1249 change is very recent, I think we can savely change the header here. 
And, given that this is another option, I personally would suggest parameters for
`checkForCoupling`  and `checkForConservedQuantities`
which by default are false but can be turned on.
And probably change the outputs to `argout`, explaining the order of the outputs in the documentation.
Whyt do you think?

Reply:
@tpfau I think for general users when they call the function, they would like everything to be checked, so it seems more convenient to me to activate all additional functionalities using only one flag. When you added the `checkCoupling` flag, I guess it was more for backward compatibility, i.e., to keep the same function outputs when it is not turned on, correct?